setup.py

```diff
@@ -28,7 +28,7 @@ def run(self):
 
 setup(
     name='XlsxWriter',
-    version='1.0.5',
```

Comment:
This shouldn't be changed in a commit. Version changes are done by the release script.